src/AmqpBundle/Resources/config/sandbox_services.yml

```diff
@@ -0,0 +1,6 @@
+parameters:
+    m6_web_amqp.exchange.class: 'M6Web\Bundle\AmqpBundle\Sandbox\NoopExchange'
+    m6_web_amqp.queue.class: 'M6Web\Bundle\AmqpBundle\Sandbox\NoopQueue'
+    m6_web_amqp.connection.class: 'M6Web\Bundle\AmqpBundle\Sandbox\NoopConnection'
+    m6_web_amqp.channel.class: 'M6Web\Bundle\AmqpBundle\Sandbox\NoopChannel'
```

Comment:
I treated mock classes as something related only to tests. IMHO what exists in `Tests` namespace is considered internal.
Envelope was not needed because you can use it without connection.

I can either move mock classes outside of tests or if you disagree with my imho - just add connection.